docs/source/Ingestion/sprouter.md

```diff
@@ -1,17 +1,17 @@
 # Building a graph from your data
 
-The first step to getting your first temporal graph analysis up and running is to tell Raphtory how to read your data source and how to build it into a graph. 
+The initial step to getting your first temporal graph analysis up and running is to tell Raphtory how to read your data source and how to build it into a graph. 
 
 Two classes help with this:
 
-- `Spouts` help with reading the data file and output tuples.
+- `Spouts` connect to the outside world, reading the data files and outputting individual tuples.
 - `Graph builders`, as the name suggests, convert these tuples into updates, building the graph.
 
-Once these classes are defined, they can be passed to the `stream()` or `batchLoad()` methods on the `Raphtory` object, which will use both components to build the temporal graph.  
+Once these classes are defined, they can be passed to the `stream()` or `batchLoad()` methods on the `Raphtory` object, which will use both components to build the temporal graph. The difference here is that `stream()` will launch the full pipeline and assume new data can continuously arrive, whilst `batchLoad()` will compress the Spout and Graph Builder functions together, running as fast as possible, but only on a static dataset which does not change. For all examples discussed, both will work fine!
 
-If you have downloaded the [Examples](https://github.com/Raphtory/Examples.git) folder from the installation guide previously, then the LOTR example is already set up. If not, please return there and complete this step first.  
+If you have the LOTR example already set up from the installation guide previously ([raphtory-example-lotr](https://github.com/Raphtory/Raphtory/tree/master/examples/raphtory-example-lotr)) then please continue. If not, please return there and complete this step first.  
 
-For this tutorial section we will continue to use the `raphtory-example-lotr` project and a dataset that tells us when two characters have some type of interaction in the Lord of the Rings trilogy books. The `csv` file (comma-separated values) in the examples folder can be found [here](https://github.com/Raphtory/Examples/blob/0.5.0/raphtory-example-lotr/resources/lotr.csv). Each line contains two characters that appear in the same sentence, along with which sentence they appeared in, indicated by a number (sentence count). In the example, the first line of the file is `Gandalf,Elrond,33` which tells us that Gandalf and Elrond appear together in sentence 33.  
```

Comment:
This link is to the project Raphtory/Examples, is that ok?

Reply:
Good catch I will update it.